SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorCloudLauncher.h

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Misc/Paths.h"
+#include "Logging/LogMacros.h"
+
+const FString DeploymentLauncherAbsolutePath = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectDir() / TEXT("Plugins/UnrealGDK/SpatialGDK/Binaries/ThirdParty/Improbable/Programs/DeploymentLauncher")));
```

Comment:
I'm not sure this is safe to do in global initialization, how do we know that FPaths is ready to use?

Reply:
Makes sense - it can be moved to one of the settings files?

Reply:
Moved it to the editor settings.